packages/locales/lib/human/de.json

```
@@ -734,5 +741,21 @@
   "reset_s2cells": "S2 Zellen zurücksetzen",
   "reset_spawnpoints": "Spawnpoints zurücksetzen",
   "reset_submission_cells": "Submission Zellen zurücksetzen",
-  "hisuian": "Hisui"
+  "hisuian": "Hisui",
+  "spacial_rend_range": "Raumschlag Effekt Bereich",
+  "key": "Schlüssel",
+  "ai": "KI",
+  "human": "Human",
+  "locales": "Lokalisierung",
```

```suggestion
  "locales": "Locales",
```

There is no translation for that, Lokalisierung is something different in computer science :) 
https://de.wikipedia.org/wiki/Locale